test/functional/api/v2/organization_controller_test.rb

```diff
@@ -0,0 +1,243 @@
+require 'test_helper'
+
+class Api::V2::OrganizationsControllerTest < ActionController::TestCase
+  def setup
+    @organization = taxonomies(:organization1)
+    @organization.location_ids = [taxonomies(:location1).id]
+    Rabl.configuration.use_controller_name_as_json_root = false
+  end
+
+  test "should get index" do
+    get :index, { }
+    assert_response :success
+    assert_not_nil assigns(:organizations)
+  end
+
+  test "should show organization" do
+    get :show, { :id => taxonomies(:organization1).to_param }
+    assert_response :success
+    show_response = ActiveSupport::JSON.decode(@response.body)
+    assert !show_response.empty?
```

Comment:
@eLobato sure thing.
you want me to make the same change on the location side?
(would that go into a separate PR?)